repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -65,9 +72,9 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
     } else {
       val wrappedRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
       val wrappedResponse = new HttpServletResponseWrapper(
-        servletResponse.asInstanceOf[HttpServletResponse], ResponseMode.MUTABLE, ResponseMode.PASSTHROUGH)
+        servletResponse.asInstanceOf[HttpServletResponse], ResponseMode.MUTABLE, ResponseMode.MUTABLE)
```

Comment:
We could handle this a little smarter, only going to mutable if it's a matching resource. It's one less spot where we break the stream. It won't make a huge difference though, since we tinker with headers, and the response can't actually be written till they are done.